src/mpi/romio/adio/include/adioi_fs_proto.h

```diff
@@ -49,4 +50,66 @@ extern struct ADIOI_Fns_struct ADIO_GPFS_operations;
 /* prototypes are in adio/ad_gpfs/ad_gpfs.h */
 #endif
 
+#ifdef ROMIO_DAOS
+/* prototypes are in adio/ad_daos/ad_daos.h */
+extern struct ADIOI_Fns_struct ADIO_DAOS_operations;
+
```

Comment:
that's it.  that's all the DAOS bits that should be in adioi_fs_proto.h.   Can 'handle_share' and the enum go in to ad_daos_common.h?  ah, or are these routines piercing the ADIO abstraction layer?

Reply:
hmm no, i should be able to move that.. i think i had it here before when i was doing the daos_init / finalize above the adio layer.. will fix shortly.

Reply:
so actually i had fixed that before, but im not sure why is it no showing in this PR.
https://github.com/daos-stack/mpich/blob/daos_adio/src/mpi/romio/adio/include/adioi_fs_proto.h

hmm maybe i should just recreate the PR

Reply:
are you sure you are not looking at an older version.. looking through the PR diff, i see that was already moved to ad_daos/ad_daos.h